modules/logging-enabling-loki-alerts.adoc

```diff
@@ -12,14 +12,14 @@ The `AlertingRule` CR contains a set of specifications and webhook validation de
 * If an `AlertingRule` CR includes an invalid `for` period, it is an invalid alerting rule.
 * If an `AlertingRule` CR includes an invalid LogQL `expr`, it is an invalid alerting rule.
 * If an `AlertingRule` CR includes two groups with the same name, it is an invalid alerting rule.
-* If none of above applies, an alerting rule is considered valid.
+* If none of the above applies, an alerting rule is considered valid.
 
 [options="header"]
 |================================================
 | Tenant type    | Valid namespaces for `AlertingRule` CRs
-| application    |
 | audit          | `openshift-logging`
 | infrastructure | `openshift-/\*`, `kube-/\*`, `default`
```

Comment:
This does not seem to parse well. The `\` and `/` are not meant to be in the output. To me it looks as if we can just use `openshift-*` and `kube-*`.